src/com/xilinx/rapidwright/interchange/PhysNetlistReader.java

```diff
@@ -786,6 +790,34 @@ else if (isPlaced != isCellPlaced) {
 
             }
         }
+    }
+
+    private static void postProcess(Design design) {
+        final Series series = design.getDevice().getSeries();
 
+        if (series == Series.UltraScalePlus || series == Series.UltraScale) {
+            // To be consistent with Vivado DCPs, remove all intra-site routing for
+            // SRST* pins tied to ground on these series of devices.
+            // (Note: this condition is necessary for {@link DesignTools#createCeSrRstPinsToVCC()})
+            for (SiteInst si : design.getSiteInsts()) {
+                if (!Utils.isSLICE(si)) {
+                    continue;
+                }
+                for (String sw : new String[]{"RST_ABCDINV_OUT", "RST_EFGHINV_OUT"}) {
```

Comment:
Perhaps consider moving the `new String[]` to outside the loop since its contents are constant.